app/components/publications/publication/proofs/proof-info-panel.hbs

```diff
@@ -0,0 +1,118 @@
+          <h4 class="auk-panel__title">
+            {{t "proof-info"}}
+          </h4>
+        </Auk::Toolbar::Item>
+      </Auk::Toolbar::Group>
+      <Auk::Toolbar::Group @position="right">
+        <Auk::Toolbar::Item>
+          <Auk::Button
+            @skin="tertiary"
+            @icon="pencil"
+            @loading={{this.loadStatus.isRunning}}
+            @layout="icon-only"
+            {{on "click" this.openEditingPanel}}
+          />
+        </Auk::Toolbar::Item>
+      </Auk::Toolbar::Group>
+    </Auk::Toolbar>
+  </Auk::Panel::Header>
+
+  {{#if this.isEditing}}
+    <Auk::Panel::Body>
+      <div class="auk-o-grid">
+        <div class="auk-o-grid-col-3">
+          <div class="auk-form-group">
+            <Auk::Label>{{t "publication-due-date"}}</Auk::Label>
+            <Auk::Datepicker
+              @date={{this.dueDate}}
+              @onChange={{this.setPublicationDueDate}}
+            />
+            {{#if
+              (and
+                (not this.publicationStatus.isPublished)
+                (is-after this.dueDate precision="days")
+              )
+            }}
+              <Auk::FormHelpText
+                @text={{t "date-expired"}}
+                @skin="warning"
+                @icon="alert-triangle"
+              />
+            {{/if}}
+          </div>
+        </div>
+        <div class="auk-o-grid-col-3">
+          <div class="auk-form-group">
+            <Auk::Label>
+              {{t "proofprint-corrector"}}
+            </Auk::Label>
+            <Auk::Input
+              @value={{mut this.proofPrintCorrector}}
+            />
+          </div>
+        </div>
+      </div>
+    </Auk::Panel::Body>
+    <Auk::Panel::Footer>
+      <Auk::Toolbar>
+        <Auk::Toolbar::Group @position="left">
+          <Auk::Toolbar::Item>
+            <Auk::Button
+              @skin="borderless"
+              @disabled={{this.save.isRunning}}
+              {{on "click" this.closeEditingPanel}}
+            >
+              {{t "cancel"}}
+            </Auk::Button>
+          </Auk::Toolbar::Item>
+        </Auk::Toolbar::Group>
+        <Auk::Toolbar::Group @position="right">
+          <Auk::Toolbar::Item>
+            <Auk::Button @skin="primary"  @loading={{this.save.isRunning}} {{on "click" (perform this.save)}}>
+              {{t "save"}}
+            </Auk::Button>
+          </Auk::Toolbar::Item>
+        </Auk::Toolbar::Group>
+      </Auk::Toolbar>
+    </Auk::Panel::Footer>
+  {{else}}
+    <Auk::Panel::Body>
+      <div class="auk-o-grid">
+        <div class="auk-o-grid-col-3 auk-content">
+          <h4 class="auk-u-m-0">{{t "publication-due-date"}}</h4>
+          <p class="auk-u-m-0">
+            {{#if @publicationSubcase.dueDate}}
+              {{moment-format @publicationSubcase.dueDate "DD-MM-YYYY"}}
+            {{else}}
+              -
+            {{/if}}
+          {{#if
+            (is-after @publicationSubcase.dueDate precision="days")
```

Comment:
In edit-mode is er een extra conditie dat de publicatie als status niet 'Gepubliceerd' heeft